test/saithrift/tests/sail3.py

```diff
@@ -3514,6 +3517,9 @@ def runTest(self):
 
         ### create route
         sai_thrift_create_route(self.client, vr_id, addr_family, ip_addr1_subnet, ip_mask1, nhop1)
+        sai_thrift_create_route(self.client, vr_id, addr_family, ip_addr_subnet, ip_mask, rif_id1 )
```

Comment:
Thanks.
Please remove the redundant space at the end of line 3520.
Please also remove the local routes in cleanup stage.

Reply:
updated cleanup and remove the redundant space